src/client/components/SearchMessagesWords/SearchMessages.css

```diff
@@ -0,0 +1,23 @@
+.lista {
```

Comment:
CSS should be "scoped" to prevent global styles. In this case your container is `searchBox` so this should become `.searchBox .lista`.

Also, `lista` should probably change the name to `results` or something similar.